job-service-acceptance-tests/src/test/java/com/hpe/caf/jobservice/acceptance/JobServiceDatabaseUtil.java

```diff
@@ -130,13 +131,13 @@ public static void assertJobDependencyRowsExist(final String jobId, final String
             st.setString(1, jobId);
             final ResultSet jobTaskDataRS = st.executeQuery();
             jobTaskDataRS.next();
-            Assert.assertEquals(jobTaskDataRS.getString(1), jobId);
-            Assert.assertEquals(jobTaskDataRS.getString(2), BatchWorkerConstants.WORKER_NAME);
-            Assert.assertEquals(jobTaskDataRS.getInt(3), BatchWorkerConstants.WORKER_API_VERSION);
-            Assert.assertTrue(jobTaskDataRS.getBytes(4).length > 0);
-            Assert.assertEquals(jobTaskDataRS.getString(5), batchWorkerMessageInQueue);
-            Assert.assertEquals(jobTaskDataRS.getString(6), exampleWorkerMessageOutQueue);
-            Assert.assertTrue(!jobTaskDataRS.next());
+            Assert.assertEquals(jobTaskDataRS.getString(1), jobId, "Job "+jobId+" does not exist in job_task_data table");
+            Assert.assertEquals(jobTaskDataRS.getString(2), BatchWorkerConstants.WORKER_NAME, "Worker "+BatchWorkerConstants.WORKER_NAME+" does not exist in job_task_data table");
+            Assert.assertEquals(jobTaskDataRS.getInt(3), BatchWorkerConstants.WORKER_API_VERSION, "Worker API Version "+BatchWorkerConstants.WORKER_API_VERSION+" does not exist in job_task_data table");
+            Assert.assertTrue(jobTaskDataRS.getBytes(4).length > 0, jobTaskDataRS.getBytes(4)+" array is empty");
+            Assert.assertEquals(jobTaskDataRS.getString(5), batchWorkerMessageInQueue, "batchWorkerMessageInQueue: "+jobTaskDataRS.getString(5)+" is not matching as: "+batchWorkerMessageInQueue);
+            Assert.assertEquals(jobTaskDataRS.getString(6), exampleWorkerMessageOutQueue, "exampleWorkerMessageOutQueue: "+jobTaskDataRS.getString(6)+" is not matching as: "+exampleWorkerMessageOutQueue);
+            Assert.assertTrue(!jobTaskDataRS.next(), "Duplicate result for Job "+jobId+" should not have been existed in job_task_data table");
```

Comment:
Reword this as it does not make sense:
```suggestion
            Assert.assertTrue(!jobTaskDataRS.next(), "Duplicate result for Job "+jobId+" should not exist in job_task_data table");
```